pkg/kubelet/container_bridge.go

```diff
@@ -43,15 +43,16 @@ func createCBR0(wantCIDR *net.IPNet) error {
 		glog.Error(err)
 		return err
 	}
-	// restart docker
+	// Stop docker so that babysitter process can restart it again with proper configurations and
+	// checkpoint file
 	// For now just log the error. The containerRuntime check will catch docker failures.
 	// TODO (dawnchen) figure out what we should do for rkt here.
 	if util.UsingSystemdInitSystem() {
-		if err := exec.Command("systemctl", "restart", "docker").Run(); err != nil {
+		if err := exec.Command("systemctl", "stop", "docker").Run(); err != nil {
```

Comment:
> On containerVM image today, the init system can not do health-check by itself, we introduced monit first, later supervisord to help with this as the extension of init system on KNode.

This is exactly why I'm confused. It makes me believe we have a special case where "stop" is talking to pid1 (sysvinit) which isn't like the other init systems, and supervisord is saving us because it doesn't understand "stop".